ivy/functional/frontends/jax/numpy/searching_sorting.py

```diff
@@ -90,3 +98,26 @@ def extract(condition, arr):
     if condition.dtype is not bool:
         condition = condition != 0
     return arr[condition]
+
+
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_like_without_promotion
+def sort(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    axis: int = -1,
+    descending: bool = False,
+    stable: bool = True,
+    out: Optional[ivy.Array] = None,
+):
+    if axis == 1:
```

Comment:
kindly debug your code from here as the difference is very minimal as the assertion error is `[1 0] != [0 1]`, with the original framework, you are missing some logical implementation while implementing this

Reply:
No, I didn't forget but I am still trying to solve the problem every day.